### PR TITLE
Added UI schema and frontend implementation

### LINKED ISSUE #2: Add server support for uiSchema
On server side in same directory with `action.yaml` can be optional file `ui-schema.yaml`

Which can contains custom widgets for defined fields. And if this file exists provide extra field in action response

More info:

https://github.com/launchrctl/launchr/pull/30/files
https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/uiSchema/

So here can be returned uiSchema field which contains schema of uiSchema file

https://github.com/launchrctl/web/blob/main/server/openapi.yaml#L39

### PR DESCRIPTION
**Pull request by iberdinsky-skilld**
Closes #2

### CODE REVIEW

File: server/api.go [hunk 1]
```diff
@@ -157,11 +159,33 @@ func apiActionFull(baseURL string, a *action.Action) ActionFull {
...
+    } else {
+      panic(err)
+    }
+  } else {
+    var data interface{}
+    err = yaml.Unmarshal(yamlData, &data)
+    if err != nil {
+      panic(err)
+    }
+    resultMap, _ = data.(map[string]interface{})
```
Comment (iignatevich) [inline server/api.go:180]: would be great to check if assertion was ok

File: server/api.go [hunk 2]
```diff
@@ -157,11 +159,33 @@ func apiActionFull(baseURL string, a *action.Action) ActionFull {
...
+  }
+
 	return ActionFull{
 		ID:          a.ID,
 		Title:       def.Title,
 		Description: def.Description,
 		JSONSchema:  jsonschema,
+		UISchema:    &resultMap,
```
Comment (iignatevich) [inline server/api.go:188]: no need to pass map by &

File: server/api.go [hunk 3]
```diff
@@ -157,11 +159,33 @@ func apiActionFull(baseURL string, a *action.Action) ActionFull {
 	jsonschema := a.JSONSchema()
 	jsonschema.ID = fmt.Sprintf("%s/actions/%s/schema.json", baseURL, url.QueryEscape(a.ID))
 	def := a.ActionDef()
+
+
+	var resultMap map[string]interface{}
+
+  yamlData, err := os.ReadFile(fmt.Sprintf("%s/ui-schema.yaml", a.Dir()))
```
Comment (iignatevich) [inline server/api.go:166]: fix indent please, command `make lint` will do it for you

Comment (iberdinsky-skilld) [reply server/api.go]: fixed

**Pull request merged**